lib/constants.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
-const VERSION = '1.78.0';
+const VERSION = '1.78.1';
```

Comment:
Feel free to bump the minor version instead of just the patch version. This is more than a bugfix, a feature rather: `1.79.0`. Bumps the version for the next release which might include more.

Reply:
@luchaos, I just tought that this change is so small (and only concerns preformance improvement), that I felt increasing minor version just for that would be too much.
Addressing above comment about other possibilities for improvement (on player, or game page), I can include it in this PR as well (and then bump the minor version). Just the problem with active players widget was in my opinion so heavily impacting performance, that I focused on that one.

Reply:
@luchaos oh, I just noticed that achievement, game, and user images on other pages are already lazy loaded! You did that last year in april, here: [https://github.com/RetroAchievements/RAWeb/commit/92e4f03ba3899abe733e7c415ae6185fadd1b266](https://github.com/RetroAchievements/RAWeb/commit/92e4f03ba3899abe733e7c415ae6185fadd1b266)

So I guess there is nothing more to do, and active players widget is the thing, which indeed needs optimization.
Should I bump minor version before merging, or it can go like that?

Reply:
oh, you're right! i messed up my search and didn't see those 😅

Reply:
Going for the minor version is fine - it's not about the size but the type of change. I usually bump the minor version before release but you can do so in this PR, too, or just leave it as is. Both is fine imo.